configs/ubuntu16_04_clang/2.0/BUILD

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
Lets create a 1.1 ubuntu16_04_clang version not a 2.0

Reply:
Done.